script.js

```diff
@@ -0,0 +1,127 @@
+let first = document.getElementById('journey');
+let second = document.getElementById('discount');
+
+/* travel planner */
+
+first.addEventListener('click', () => {
+
+    let objectOfCountries = {
+        Ukraine: 500,
+        Italia: 1500,
+        Tailand: 1000, 
+        Poland: 699,
+        Grecce: 1099,
+        Moldova: 299
+    }   
+    
+    let country = prompt('Введіть бажану країну для подорожі - Ukraine, Italia, Tailand, Poland, Grecce or Moldova:');
+    let money = prompt('Введіть наявну суму подорожі:');
+
+    function getResult(country, money) {
+
+        if (Number(money) >= objectOfCountries[country]) {
+            alert(`Рівень бюджету достатній для подорожі в ${country}`);
+
+        } else if (Number(money) < objectOfCountries[country] && Number(money) > 298) {
```

Comment:
вдруг цена изменится?
код должен уметь подстраиваться под изменения
```suggestion
        } else if (Number(money) < objectOfCountries[country] && Number(money) => objectOfCountries["Moldova"]) {
```